sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/util/ZipFileCreatorTest.java

```diff
@@ -0,0 +1,23 @@
+package com.mercedesbenz.sechub.wrapper.xray.util;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ZipFileCreatorTest {
+
+    ZipFileCreator creatorToTest;
+
+    @BeforeEach
+    void beforeEach() {
+        creatorToTest = new ZipFileCreator();
+    }
+
+    @Test
+    void saveInputStreamToZipFile_throws_nullPointerException() {
+        /* execute + test */
```

Comment:
Please remove the NPE test or provide illegal argument exception and check messages etc.